frontend/src/constants/tour-steps.tsx

```diff
@@ -0,0 +1,55 @@
+import type { StepType } from '@reactour/tour';
+import { T } from '@transifex/react';
+
+const PADDING = 4;
+const SPACE = 35 + PADDING;
+
+const TOUR_STEPS: StepType[] = [
```

Comment:
This will have to be created as a function and execute it inside the component for it to change when we change the language. Please check the readme. I know that is a bit annoying the process but it wont work otherwise